backend/pkg/api/data_access/notifications.go

```diff
@@ -146,23 +155,12 @@ func (d *DataAccessService) GetDashboardNotifications(ctx context.Context, userI
 	defaultColumns := []t.SortColumn{
```

Comment:
**thought:** Since the default column entries only require those that uniquely identify a row you don't need to add the dashboard and group names to it.
However if you change it this way you would need the `offset` if again.

Both ways work for me but consider that the query could be faster with less WHERE/ORDER BY.